backend/pricing/pricing.go

```diff
@@ -234,14 +254,15 @@ func GetStripePrices(stripeClient *client.API, productTier backend.PlanType, int
 
 	sessionsLookupKey := string(ProductTypeSessions)
 	membersLookupKey := string(ProductTypeMembers)
+	errorsLookupKey := string(ProductTypeErrors)
 
 	priceListParams := stripe.PriceListParams{}
-	priceListParams.LookupKeys = []*string{&baseLookupKey, &sessionsLookupKey, &membersLookupKey}
+	priceListParams.LookupKeys = []*string{&baseLookupKey, &sessionsLookupKey, &membersLookupKey, &errorsLookupKey}
 	prices := stripeClient.Prices.List(&priceListParams).PriceList().Data
 
 	// Validate that we received exactly 1 response for each lookup key
-	if len(prices) != 3 {
-		return nil, e.Errorf("expected 3 prices, received %d", len(prices))
+	if len(prices) != 4 {
+		return nil, e.Errorf("expected 4 prices, received %d", len(prices))
 	}
```

Comment:
not certain we should do this now, but we should keep in mind that we'll likely be adding per-logs pricing soon, so if there are constants we can make / ways to abstract this so that the per-product pricing is managed in one place, that would be great.